src/quantum/setup.py

```diff
@@ -17,7 +17,7 @@
 # This version should match the latest entry in HISTORY.rst
 # Also, when updating this, please review the version used by the extension to
 # submit requests, which can be found at './azext_quantum/__init__.py'
-VERSION = '0.16.0'
```

Comment:
Is this version just used during development? I don't know if this will trigger a release. What is the intention here? I assume that the real release will happen when the version goes to 0.17.0, but I'm not sure about the consequences of merging as is. Have you discussed it with the CLI team?

Reply:
I'll restore the 0.16.0 version number before I click "Ready for review".